ivy/functional/ivy/statistical.py

```diff
@@ -1310,3 +1315,40 @@ def einsum(
     }
     """
     return current_backend(operands[0]).einsum(equation, *operands, out=out)
+
+
+@with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
+@infer_device
+@infer_dtype
+@handle_array_function
+@outputs_to_ivy_arrays
+@inputs_to_ivy_arrays
+def nanmax(a, axis=None, keepdims=False):
```

Comment:
This should call the backend functions you defined.

Reply:
Why are you not addressing my questions directly? I have asked from the beginning about this. First, there was to follow a merged PR. I did that exactly and 8 files were changed accordingly. This resulted in the same errors as before that were not explained. While I was waiting for your response, an ivy member on discord advised to include the fron-end implementations for each framework in the backend, which I did and resulted into errors. I would like a technical review of the code and pinpointing exactly the problem. For now, I assume it's about directories and duplicity.

Reply:
My questions are specifically about the confusion I had, directly emerging from the documentation. If the questions can't be answered directly, I suggest I will be working on this tomorrow onwards and debug it with others who will be willing to answer questions directly.